react/features/base/app/components/BaseApp.native.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import React from 'react';
+
+import { DialogContainer } from '../../dialog';
+
+import AbstractBaseApp from './AbstractBaseApp';
+
+/**
+ * Base (abstract) class for main App component.
+ */
+export default class BaseApp extends AbstractBaseApp<*> {
+    /**
+     * Initializes a new {@code BaseApp} instance.
+     *
+     * @inheritdoc
+     */
+    constructor() {
+        super();
+
+        this._renderDialogContainer = this._renderDialogContainer.bind(this);
+    }
+
+    _renderDialogContainer: () => React$Element<*>
+
+    /**
+     * Renders the platform specific dialog container.
+     *
+     * @returns {React$Element}
+     */
+    _renderDialogContainer() {
+        return (
+            <DialogContainer />
+        );
+    }
```

Comment:
Another idea I can throw around because I'm not doing the actual work is having the web implementation of DialogContainer just wrap itself in the theme, respecting a passed in theme if needed otherwise default to dark. This might mean also abstracting away atlaskit/theme in a wrapper so it can work on web and native.